#!/usr/bin/env python
# multigoal_planner configuration

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, bool_t, int_t

gen = ParameterGenerator()

gen.add("publish_human_plans", bool_t, 0, "Wheter to pulish human plans for visualization.", True)
gen.add("publish_human_poses", bool_t, 0, "Wheter to pulish poses of human plans for visualization.", True)
gen.add("publish_potential", bool_t, 0, "Wheter to pulish potential of calculated during planning for visualization.", False)
gen.add("poses_z_reduce_factor", int_t, 0, "The factor by which to reduce z value of poses for visualization.", 100, 1, 200)

gen.add("restore_defaults", bool_t, 0, "Restore to the original configuration", False)

exit(gen.generate('multigoal_planner', "multigoal_planner", "MultiGoalPlanner"))
